src/ophyd_async/epics/pvi/pvi.py

```diff
@@ -296,3 +306,30 @@ async def fill_pvi_entries(
     # We call set name now the parent field has been set in all of the
     # introspect-initialized devices. This will recursively set the names.
     device.set_name(device.name)
+
+
+def pre_initialize_blocks(
+    device: Device, included_optional_fields: Optional[Tuple[str, ...]] = None
```

Comment:
Tuple is not mutable so can simplify
```suggestion
    device: Device, included_optional_fields: Tuple[str, ...] = ()
```